docs/02_types/interfaces.md

```diff
@@ -5,4 +5,194 @@ path: /docs/types/interfaces
 
 # Interfaces
 
-Documentation coming soon
+Interfaces are an abstract type which may be implemented by object types.
+
+An interface has fields, but it’s never instantiated. Instead, objects may
+implement interfaces, which makes them a member of that interface. Also, fields
+may return interface types. When this happens, the returned object may be any
+member of that interface.
+
+For example, let's say a `Customer` (interface) can either be an `Individual`
+(object) or a `Company` (object). Here's what that might look like in the
+[GraphQL Schema Definition Language](https://graphql.org/learn/schema/#type-language) (SDL):
+
+```graphql
+interface Customer {
+  name: String!
+}
+
+type Company implements Customer {
+  employees: [Individual!]!
+  name: String!
+}
+
+type Individual implements Customer {
+  employed_by: Company
+  name: String!
+}
+
+type Query {
+  customers: [Customer!]!
+}
+```
+
+Notice that the `Customer` interface requires the `name: String!` field. Both
+`Company` and `Individual` implements that field so that they can implement the
+`Customer` interface.
+
+When querying, you can select the fields on an interface:
+
+```graphql
+query {
+  customers {
+    name
+  }
+}
+```
+
+Whether the object is a `Company` or an `Individual`, it doesn’t matter – you
+still get their name. If you want some object-specific fields, you can query
+them with an inline fragment, for example:
+
+```graphql
+query {
+  customers {
+    name
+    ... on Individual {
+      company {
+        name
+      }
+    }
+  }
+}
+```
+
+Interfaces are a good choice whenever you have a set of objects that are used
+interchangeably, and they have several significant fields in common. When they
+don’t have fields in common, use a [Union](/docs/types/unions) instead.
+
+## Defining interfaces
+
+Interfaces are defined using the `@strawberry.interface` decorator:
+
+```python+schema
+import strawberry
+
+@strawberry.interface
+class Customer:
+    name: str
+---
+interface Customer {
+  name: String!
+}
+```
+
+> Note: Interface classes should never be instantiated directly.
+
+## Implementing interfaces
+
+To define object type that implement an interface the type must inherit from the
```

Comment:
`define object type that implement` -> `define an object type that implements`

Also, add comment after `interface`

Reply:
Done